solutions/4-getting-started-js/countBy.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-var */
+
+import { groupBy } from './groupBy.js'
+
+var countBy = function (groupingFunction, arr) {
+  return Object.entries(groupBy(groupingFunction, arr)).reduce((item, [key, val]) => {
```

Comment:
Good, but restructure your formatting a little. The line is too long like that and hard to read.